apps/service_profile/config/test.exs

```diff
@@ -20,3 +20,5 @@ config :service_profile, Profile.Application,
     ],
     dispatcher: Brook.Dispatcher.Noop
   ]
+
+config :service_persist, Persist.MetricsReporter, port: 9574
```

Comment:
Why is this a different port for test and integration whileas broadcast uses the same port for both?

Reply:
b/c i screwed up. this port is in use by `persist`

Reply:
good catch